### PR TITLE
Documented prior/on-going works

### PR DESCRIPTION
**Pull request by ralph-lange**
- Added some sentences about prior and on-going works
- Removed Links column in Actions section
- Also renamed this section to Analyses and Experiments
- Renamed W-MW to W-COMM

### CODE REVIEW

File: articles/embedded.md
Comment (ralph-lange) [inline articles/embedded.md]: I propose to replace this architecture diagram with the more abstract diagram by Borja, i.e. with micro_ros_arch_components.png. The current diagram is rather fine-grained given the abstract discussion above where as Borjas diagram could be a good starting point to (1.) discuss the desired feature set for an embedded ROS 2 stack and (2.) the implications and dependencies with regard to the lower layers.

File: articles/embedded.md
Comment (BorjaOuterelo) [inline articles/embedded.md]: `micro-XRCE-DDS` should be `Micro XRCE-DDS` to match the eProsima product name.

File: articles/embedded.md
Comment (BorjaOuterelo) [inline articles/embedded.md]: `micro-XRCE` should be changed for `Micro XRCE-DDS`

File: articles/embedded.md
Comment (BorjaOuterelo) [inline articles/embedded.md]: I would remark that this project has been abandoned and it has not been updated for quite a long time.

File: articles/embedded.md
Comment (BorjaOuterelo) [reply articles/embedded.md]: I am fine with including that one. The raw diagram file is in OFERA's git, just in case some modifications are needed.
On the other hand, I think we should also keep what our design of the architecture is.
I think even having a more concrete design, discussions on the features and their implications will arise. Maybe this is not the place for that kind of diagram, but we should get it exposed to the community.

**Review by BorjaOuterelo — CHANGES_REQUESTED** [review]

File: articles/embedded.md
Comment (BorjaOuterelo) [inline articles/embedded.md:34]: Why we switched it to COMM? Is MW not more suitable?

**Review by BorjaOuterelo — DISMISSED** [review]

File: articles/embedded.md
Comment (ralph-lange) [reply articles/embedded.md:34]: The corresponding wish and question cover all aspects, from physical communication technologies up to the middleware. For this reason and given the fact that COMM is commonly known for 'communication', I propose to change MW to COMM.

File: articles/embedded.md
Comment (ralph-lange) [reply articles/embedded.md]: We already have it exposed to the community at micro-ros.com. During the course of the discussion on this article, we probably reach the point where we can put the detailed diagram on the table. Yet, for the start, I would prefer a more abstract view focussed on the API features.

Comment (ralph-lange) [reply articles/embedded.md]: @BorjaOuterelo I just added your diagram and a link to the other diagram on micro-ros.com. I propose to replace 'rclc' with 'rclc/rclcpp' and to fix the name of the middleware. Yet, I did not find the sources in our gitlab repo. Can you point me to the corresponding file or change it yourself, please?

File: articles/embedded.md
Comment (ralph-lange) [inline articles/embedded.md:93]: @BorjaOuterelo I just added your diagram and a link to the other diagram on micro-ros.com. I propose to replace 'rclc' with 'rclc/rclcpp' and to fix the name of the middleware. Yet, I did not find the sources in our gitlab repo. Can you point me to the corresponding file or change it yourself, please?

Comment (BorjaOuterelo) [reply articles/embedded.md:93]: @ralph-lange In this file you can find all the diagrams sources: [https://gitlab.com/micro-ROS/documentation/blob/f26df8af4c2066e7ee4fbb4551252aa788d75450/architecture/diagrams/micro-ROS_architecture.svg]()
For editing it, I recommend you [https://www.draw.io]().
That online tool allows you to export selected parts of the diagram to different formats.
If you need any help with it ask me please.

**Review by vmayoral — DISMISSED** [review]

**Review by iluetkeb — APPROVED** [review]

**Comment by ralph-lange** [comment]
@vmayoral , @BorjaOuterelo  I fixed the urls. Could you please check and approve, so that I can merge this PR.

**Review by BorjaOuterelo — APPROVED** [review]

**Comment by BorjaOuterelo** [comment]
@ralph-lange, Just approved I will rebase the other PR once this is merged, as I also changed the urls on the other PR

**Pull request merged**
